.gitignore

```diff
@@ -21,9 +21,12 @@ hs_err_pid*
 *.iml
 
 ## Directory-based project format:
-.idea/
+.idea/*
```

Comment:
hm, honestly, I do not commit from IDEA, because of that I have not seen a such issue :) also, I thought the IDEA does not really change these files very often. Maybe it is a bug in IDEA?
Also my setup is:
git version 2.10.0
IDEA 2017.1.4